readthedocs/builds/migrations/0057_migrate_timestamp_fields.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.2.9 on 2024-02-01 20:29
+
+import datetime
+
+from django.db import migrations
+from django_safemigrate import Safe
+
+
+def migrate(apps, schema_editor):
+    """
+    Migrate the created and modified fields of the Version model to have a non-null value.
+
+    This date corresponds to the release date of 5.6.5,
+    when the created field was added to the Version model
+    at https://github.com/readthedocs/readthedocs.org/commit/d72ee6e27dc398b97e884ccec8a8cf135134faac.
+    """
+    Version = apps.get_model("builds", "Version")
+    date = datetime.datetime(2020, 11, 23, tzinfo=datetime.timezone.utc)
```

Comment:
Shouldn't we use a date that it's obvious it's wrong to avoid confusions here? Like `1999-01-01` or similar.

Reply:
I think that would add more confusion, for us and users (if we expose this field).